<ui version="4.0" >
 <class>OperatorCardDialog</class>
 <widget class="QDialog" name="OperatorCardDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>395</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../icons/resources.qrc" >:/32/user.png</iconset>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="5" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>5</hsizetype>
         <vsizetype>5</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize" >
        <size>
         <width>128</width>
         <height>128</height>
        </size>
       </property>
       <property name="title" >
        <string>Photo:</string>
       </property>
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item row="0" column="0" >
         <widget class="QLabel" name="photolabel" >
          <property name="maximumSize" >
           <size>
            <width>128</width>
            <height>128</height>
           </size>
          </property>
          <property name="lineWidth" >
           <number>0</number>
          </property>
          <property name="text" >
           <string>No Photo:</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="phototoolButton" >
         <property name="text" >
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="6" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="addpushButton" >
       <property name="text" >
        <string>Add</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2" >
       <property name="text" >
        <string>Cancel</string>
       </property>
       <property name="autoDefault" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0" >
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="0" column="0" >
      <widget class="QLabel" name="label_7" >
       <property name="text" >
        <string>Current password:*</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QLineEdit" name="newpasslineEdit" >
       <property name="echoMode" >
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="label_9" >
       <property name="text" >
        <string>Controll password:*</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="curpasslineEdit" >
       <property name="echoMode" >
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QLineEdit" name="controlpasslineEdit" >
       <property name="echoMode" >
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_8" >
       <property name="text" >
        <string>New password:*</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label_6" >
       <property name="text" >
        <string>Adress:*</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="adresslineEdit" >
       <property name="minimumSize" >
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label_4" >
       <property name="text" >
        <string>Phone:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="phonelineEdit" >
       <property name="minimumSize" >
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5" >
       <property name="text" >
        <string>Mobile:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="mobilelineEdit" >
       <property name="minimumSize" >
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Born:*</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="borndateEdit" >
       <property name="date" >
        <date>
         <year>1970</year>
         <month>1</month>
         <day>1</day>
        </date>
       </property>
       <property name="calendarPopup" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Full name:*</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="namelineEdit" >
       <property name="minimumSize" >
        <size>
         <width>250</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Personal number:*</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="psnlineEdit" >
       <property name="minimumSize" >
        <size>
         <width>50</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>namelineEdit</tabstop>
  <tabstop>psnlineEdit</tabstop>
  <tabstop>borndateEdit</tabstop>
  <tabstop>phonelineEdit</tabstop>
  <tabstop>mobilelineEdit</tabstop>
  <tabstop>adresslineEdit</tabstop>
  <tabstop>curpasslineEdit</tabstop>
  <tabstop>newpasslineEdit</tabstop>
  <tabstop>controlpasslineEdit</tabstop>
  <tabstop>phototoolButton</tabstop>
  <tabstop>addpushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
 </tabstops>
 <resources>
  <include location="../icons/resources.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>OperatorCardDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>628</x>
     <y>384</y>
    </hint>
    <hint type="destinationlabel" >
     <x>464</x>
     <y>352</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
